packages/lib-utils/src/components/list-view/ListView.stories.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import '@patternfly/react-core/dist/styles/base.css';
+import { Td } from '@patternfly/react-table';
+import type { ComponentStory, ComponentMeta } from '@storybook/react';
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+
+import ListView from './ListView';
+
+export default {
+  title: 'ListView',
+  component: ListView,
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
+} as ComponentMeta<typeof ListView>;
```

Comment:
Why do we need a force cast on a [default stories metadata export](https://storybook.js.org/docs/react/writing-stories/introduction#default-export)?

This does not ensure that given object conforms to `ComponentMeta` interface, it just force casts it to it.

```ts
type A = { a: string };
export default {} as A; // passes

// vs.

const x: A = {}; // fails
export default x;
```

Update: now I see that official Storybook docs actually recommend this approach (which is weird).

I'd suggest to make it type-safe, otherwise the `as` force cast is basically redundant.

```ts
const meta: ComponentMeta<typeof ListView> = {
title: 'ListView',
  component: ListView,
  argTypes: {
    backgroundColor: { control: 'color' },
  },
};

export default meta;
```